lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -5485,7 +5486,11 @@ void ScriptContext::RegisterPrototypeChainEnsuredToHaveOnlyWritableDataPropertie
             Output::Print(_u("\n\n"));
 
             // If in verbose mode, dump data for each FunctionBody
-            if (Configuration::Global.flags.Verbose && rejitStatsMap != NULL)
+            if (
+#if defined(FLAG) || defined(FLAG_REGOVR_EXP)
```

Comment:
Do you really need this here, this whole block is under #ifdef REJIT_STATS which is defined if ENABLE_DEBUG_CONFIG_OPTIONS is defined in which case FLAGNR(Verbose) is also defined